### DISCUSSION TITLE
status of GUI applications

### DISCUSSION THREAD

**Discussion by wehlgrundspitze** [microsoft/Qcodes#3567] [Ideas]
What is the current status on GUI applications based QCodes - kind of an open source version of labber (https://www.keysight.com/de/de/products/software/application-sw/labber-software.html) or sweep-me (https://sweep-me.net). I saw there is a git repository started that, but is not maintained any more: https://github.com/QCoDeS/QCoDeS-QtUI

**Comment by jenshnielsen** [comment]
That repository has a rather more general name than the actual code in there. It was never the intention for that to be a full gui for QCoDeS. I have archived the repo and added a readme to clarify that an that it is no longer maintained.

It is not a priority for us to develop a GUI and we do not see a GUI as the best way of developing a measurement framework. That being said we are happy to discuss GUI development with anyone that are interested in taking on such a project
